README.md

```diff
@@ -20,16 +20,24 @@ ProfileFolio simplifies professional profile management by generating customized
 <details>
 <summary><strong>Install Manually</strong></summary>
 
-- Open Mongodb atlas in localhost:27017
-  Create `profileFolio` db and
+> [!IMPORTANT]  
+> These installation steps assume you already have Go, MongoDB and NodeJS installed on your machine.
+> * Node JS - https://nodejs.org/en
+> * GO      - https://go.dev/doc/install
+> * MongoDB - https://www.mongodb.com/try/download/shell
+
 
-1.  Add this sample data in collection `users`
+ Open Mongodb atlas in localhost:27017
+  Create `profileFolio` db and
 
+**1.  Add this sample data in collection `users`**
+___
 ```json
 {
   "_id": { "$oid": "667b885950aa1b4215433ff4" },
   "basics": {
     "name": "Jane Smith",
+    "username": "jsmith49",
     "label": "Software Engineer",
```

Comment:
Have you tested this in local? I guess it will fail , there is no resumes section in the user details json

Reply:
I have missed this will update both the readme.md and the users.json